<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameZone</class>
 <widget class="QDialog" name="GameZone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(10, 10, 10, 150);</string>
  </property>
  <widget class="QWidget" name="WG_Msg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>291</width>
     <height>731</height>
    </rect>
   </property>
   <widget class="QLabel" name="LB_Name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_Sign">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>签名</string>
    </property>
   </widget>
   <widget class="QWidget" name="WG_GameHistory" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="WG_Boast" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>271</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="TW_Boast">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>331</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TB_SPZone">
      <attribute name="title">
       <string>专区消息</string>
      </attribute>
      <widget class="QLineEdit" name="LE_SendContent">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="TB_HallZone">
      <attribute name="title">
       <string>大厅消息</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="PB_HeadIcon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="WG_ZoneOperation" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>721</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QLabel" name="LB_GameName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3ds Light</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>游戏名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_BackToHall">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>返回大厅</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="WG_Rooms" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>721</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QScrollArea" name="SA_Rooms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>701</width>
      <height>521</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>699</width>
       <height>519</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="WG_RealTimeRooms" native="true"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>557</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>711</width>
     <height>51</height>
    </rect>
   </property>
   <widget class="QPushButton" name="PB_CreateRoom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(71, 71, 71);</string>
    </property>
    <property name="text">
     <string>创建房间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_FastMatch">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(71, 71, 71);</string>
    </property>
    <property name="text">
     <string>快速匹配</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_NofRooms">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
